src/environment.js

```diff
@@ -147,8 +152,7 @@ var Environment = Obj.extend({
                 }
 
                 // Resolve name relative to parentName
-                if (parentName && (name.indexOf("./") == 0 ||
-                                   name.indexOf("../") == 0)) {
```

Comment:
I know there is some discussion about this in other issues, but I still don't understand. Won't this break all templates that extend other ones (or includes)? Any includes in that template will now be relative by default instead of absolute.